warehouse/models/final/schema.yml

```diff
@@ -104,7 +104,7 @@ models:
         description: "Anual equivalent CO2 emission in metric tons"
       - name: year
         description: "Survey year"
-        tests:
+        data_tests:
           - accepted_values:
```

Comment:
test to  ensure that the data in specified columns only contains certain predefined values